src/api/LoginApi.js

```diff
@@ -0,0 +1,23 @@
+import { Axios } from './Axios';
+
+export const login = async (data) => {
+  try {
+    const response = await Axios.post('/auth/login', {
+      id: data.id,
+      pw: data.pw,
+    });
+    console.log(response);
+    return response;
+  } catch (error) {
+    console.error(error);
```

Comment:
아래처럼 처리하게 된다면 편할 것 같은데 어떻게 생각하실까용?

```suggestion
    return { "status": "fail", "message": "로그인에 실패했습니다." }
```

지금은 서버쪽에서 에러 메시지에 대해 별도로 리턴을 해주지 않고, status fail 만 내려주고 있긴 한데, 이런 식으로 에러 발생시 return 을 어떻게 할꺼냐에 따라 이후 이어지는 로직이 좀 더 간결해지게 될 것 같아요.

```javascript
// ex: 위처럼 작업하게 된다면 이후 해당 함수를 사용하는 로직에서...
const result = await login(...);
if (result.status === 'fail') {
  alert(result.message);
}
```